Sources/VirtualTerminal/Platform/POSIXTerminal.swift

```diff
@@ -182,18 +186,26 @@ internal final actor POSIXTerminal: VTTerminal {
     switch mode {
     case .raw:
       // Disable canonical mode, echo, XON/XOFF, and CR to NL translation
+      #if os(Linux)
+      attr.c_lflag &= UInt32(~(UInt32(ICANON | ECHO | IXON | ICRNL)))
```

Comment:
Please sink these into swift-platform-core. We should add type cast overlay values there first.